docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -225,7 +225,7 @@ circuit.measure(qreg[5:], beta)
 
 service = QiskitRuntimeService()
 backend = service.least_busy(operational=True, simulator=False, min_num_qubits=12)
-pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+pm = generate_preset_pass_manager(backend=backend)
```

Comment:
Not that it matters, but you can still specify `optimization_level` in a pass manager

Reply:
oh yeah; I didn't mean to delete those.